File: ES_NEXT/ESD02/ESD02-DotNet/lecturedemos/Employee.js
import { Person } from './Person.js';
export default class Employee extends Person {
    constructor(_id, _name, _salary) {
        // this.ID=
        super(_id, _name);
        this.Salary = _salary;
    }
    Print() {
        return `${super.Print()} , Salary = ${this.Salary}`;
    }
}

File: C# OOP/lab7_b/lab7_b/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab7_b
{
    internal abstract class Employee
    {
        public int ID { get; set; }
        public string Name { get; set; }

        public int Salary { get; set; }

        public Client[] Clients { get; set; }


        public abstract void ShowEmployeeDetails(Client[] clients);
            
    }
}
